pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -131,15 +133,6 @@ in rec {
       };
     } ./setuptools-build-hook.sh) {};
 
-  setuptoolsCheckHook = callPackage ({ setuptools }:
```

Comment:
Is it deprecated or already non-functioning? If it is the former, we probably best keep it and add a trace that it is deprecated.